charts/incubator/zammad/values.yaml

```
@@ -0,0 +1,80 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: tccr.io/truecharts/zammad
+  tag: latest@sha256:c9c8ec07af85ca06ff2776e1bd00c70cca470c7bf9aa92c716b78283bf880f49
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsGroup: 0
+    runAsUser: 0
+
+service:
+  main:
+    enabled: false
+    ports:
+      main:
+        enabled: false
+
+zammad:
+  zammad:
+    zammad_autowizardjson: ""
+  webconcurrency:
+    webcon_instances: ""
+    webcon_sessionjobs: 0
+    webcon_schedprocesses: false
+    webcon_delayedjobs: 0
+  rails:
+    rails_trustedproxies: ['127.0.0.1', '::1']
+  redis:
+    redis_url: "http://redis.ix-redis.svc.cluster.local:6379"
```

```suggestion
    redis_url: "http://redis.ix-redis.svc.cluster.local:6379"
```
            redis_url:
              secretKeyRef:
                expandObjectName: false
                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
                key: url

---

why are we adding the redis dependency here???

---

Good call, I originally had them as config values but I believe they can be moved to right to the workload, yeah? Like below.
```yml
REDIS_URL:
  secretKeyRef:
    expandObjectName: false
    name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
    key: url```

---

Yep they can - just saw your reference [here](https://github.com/truecharts/charts/blob/7c3adc77d082b257a10afe839905ef8ae6af1b52/charts/stable/rsshub/values.yaml#L49). Should be all set with this one!